kornia/feature/histogram.py

```diff
@@ -0,0 +1,89 @@
+from typing import Tuple, List
+
+import os
+import numpy as np
+
+import torch
+import torch.nn as nn
+
+import matplotlib.pyplot as plt
+
+from PIL import Image
+from torchvision import transforms
+
+EPSILON = 1e-10
+
+
+def marginalPdf(values: torch.Tensor, bins: torch.Tensor, sigma: float = 0.7) -> Tuple[torch.Tensor, torch.Tensor]:
```

Comment:
no camelCase, use snake_case: `margina_pdf` plus add some minimal description about what the actual function does

Reply:
I guess sigma could be also a tensor that can be optimized, right ?